cmd/juju/application/deployer/deployer_test.go

```diff
@@ -82,8 +81,27 @@ func (s *deployerSuite) TestGetDeployerLocalCharm(c *gc.C) {
 	factory := s.newDeployerFactory()
 	deployer, err := factory.GetDeployer(context.Background(), cfg, s.modelConfigGetter, s.resolver)
 	c.Assert(err, jc.ErrorIsNil)
-	ch := charm.MustParseURL("local:jammy/multi-series-1")
-	c.Assert(deployer.String(), gc.Equals, fmt.Sprintf("deploy local charm: %s", ch.String()))
+	ch := "local:jammy/multi-series-1"
+	c.Assert(deployer.String(), gc.Equals, fmt.Sprintf("deploy local charm: %s", ch))
+}
+
+func (s *deployerSuite) TestGetDeployerLocalCharmPathWithSchema(c *gc.C) {
+	defer s.setupMocks(c).Finish()
+	s.expectModelGet(c)
+	cfg := s.basicDeployerConfig(corebase.MustParseBaseFromString("ubuntu@22.04"))
+	s.expectModelType()
+
+	dir := c.MkDir()
+	charmPath := testcharms.RepoWithSeries("bionic").ClonedDirPath(dir, "multi-series")
+	charmPath = fmt.Sprintf("local:%s", charmPath)
```

Comment:
I think you can just do `charmPath = "local:" + charmPath` here

Reply:
I'm not sure what we consider best practice when there is only one param to be templated. I've seen both used in the code base, but don't really have a preference

Done